### PR TITLE
Added the Fosshost

### PR DESCRIPTION
**Pull request by dragstor**

### CODE REVIEW

**Comment by swyxio** [comment]
:heavy_check_mark: Deploy Preview is done!


:hammer: Explore the source changes: cbf2caa8d43a70ad0d98bd03cfe3a28b6c53f769

:mag: Inspect the deploy log: [https://app.netlify.com/sites/svelte-community/deploys/60b8ea384739cc0008e30245](https://app.netlify.com/sites/svelte-community/deploys/60b8ea384739cc0008e30245?utm_source=github&utm_campaign=bot_dl)

:sunglasses: Browse the preview: [https://deploy-preview-453--svelte-community.netlify.app](https://deploy-preview-453--svelte-community.netlify.app?utm_source=github&utm_campaign=bot_dp)

File: whos-using-svelte/organisations/aarch64.svg [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
...},{"name":"Reggae","default":true,"gains":[0,0,0,-3.36,0,3.84,3.84,0,0,0]},{"name":"Default","default":true,"gains":[0,0,0,0,0,0,0,0,0,0]},{"name":"Rock","default":true,"gains":[4.8,2.88,-3.36,-4.8,-1.92,2.4,5.28,6.72,6.72,6.72]},{"name":"Dance","default":true,"gains":[5.76,4.32,1.44,0,0,-3.36,-4.32,-4.32,0,0]},{"name":"Techno","default":true,"gains":[4.8,3.36,0,-3.36,-2.88,0,4.8,5.76,5.76,5.28]},{"name":"Headphones","default":true,"gains":[2.88,6.72,3.36,-1.92,-1.44,0.96,2.88,5.76,7.68,8.64]},{"name":"Soft rock","default":true,"gains":[2.4,2.4,1.44,0,-2.4,-3.36,-1.92,0,1.44,5.28]},{"name":"Classical","default":true,"gains":[0,0,0,0,0,0,-4.32,-4.32,-4.32,-5.76]},{"name":"Large Hall","default":true,"gains":[6.24,6.24,3.36,3.36,0,-2.88,-2.88,-2.88,0,0]},{"name":"Full Bass","default":true,"gains":[4.8,5.76,5.76,3.36,0.96,-2.4,-4.8,-6.24,-6.72,-6.72]},{"name":"Full Treble","default":true,"gains":[-5.76,-5.76,-5.76,-2.4,1.44,6.72,9.6,9.6,9.6,10.08]},{"name":"Laptop Speakers","default":true,"gains":[2.88,6.72,3.36,-1.92,-1.44,0.96,2.88,5.76,7.68,8.64]},{"name":"Full Bass &amp; Treble","default":true,"gains":[4.32,3.36,0,-4.32,-2.88,0.96,4.8,6.72,7.2,7.2]}],"whitelist":[]})</script></svg>
```
Comment (swyxio) [inline whos-using-svelte/organisations/aarch64.svg]: wtf is this

**Review by swyxio — CHANGES_REQUESTED** [review]
could you explain what this svg does?

File: whos-using-svelte/organisations/aarch64.svg (hunk 1 above)
Comment (dragstor) [reply whos-using-svelte/organisations/aarch64.svg]: It is a 100% fair question. 

That asset was provided by the ARM Computers recently.
I've removed the JS in question and will be pushing a fix in a second.

Thanks!!!

**Comment by dragstor** [comment]
Pushed the fixed SVG for the aarch64 logo.

**Comment by benmccann** [comment]
Thanks. However, we've decided just to feature the most prominent companies using Svelte on the homepage and are no longer featuring sites submitted by the community

**Pull request closed without merging**